core/cap-0033.md

```diff
@@ -340,29 +339,29 @@ struct InnerTransactionResult
 
 #### Reserve Requirement
 No operation can take the native balance of an account below
-`(2 + numSubEntries - numSponsored) * baseReserve + liabilities.selling`.
+`(2 + numSubEntries + numSponsoring - numSponsored) * baseReserve + liabilities.selling`.
 
-#### Sponsoring Future Reserves
-When account `A` is-sponsoring-future-reserves-for account `B`, balance will be
-deducted from `A` and "stored" in any ledger entry/sub-entry for which `B` would
-need to provide reserve. Sponsoring account creation (if `B` does not yet exist)
-would deduct `2 * baseReserve` whereas sponsoring any other sub-entry would
-deduct just `baseReserve`. If `A` does not have sufficient available balance of
-native asset, then the operation fails.
+#### Sponsorship Bookkeeping
+When account `A` is-sponsoring-future-reserves-for account `B`, any reserve
+requirements that would normally accumulate on `B` will instead accumulate on
+`A` as reflected in `numSponsoring`. Both accounts are guaranteed to exist, but
```

Comment:
I like the rename of `ed` to `ing`, it reads better and describes that it is an ongoing action which was less clear in the last revision.